src/__tests__/components/MultiRowField/MultiRowField.test.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { reduxForm } from 'redux-form';
+import { createStore } from 'redux';
+import { connect, Provider } from 'react-redux';
+import { mount } from 'enzyme';
+import toJSON from 'enzyme-to-json';
+
+import MultiRowField from '../../../components/MultiRowField/MultiRowField';
+
+const initialValues = {
+    'mock-name': [
+        {
+            name: 'me',
+        },
+        {
+            name: 'you',
+        },
+    ]
+};
+
+const store = createStore(() => ({
+    form: {
+        mockForm: {
+            values: initialValues,
+        }
+    }
+}));
+
+const Form = ({ children, initialValues }) => children;
+
+const ConnectedForm = connect(
+    () => ({
+        initialValues
+    }),
+    null,
+)(
+    reduxForm({
+        form: 'mockForm',
+    })(Form)
+);
+
+const renderFakeForm = el =>
+    <Provider store={store}>
```

Comment:
This could fail since you are using a global store. If in the future, one test mutates the state of the store, it could affect the result of the remaining tests.